docker-compose.yaml

```diff
@@ -0,0 +1,8 @@
+version: '3.8'
+
+services:
+  web:
+    image: ghcr.io/sowecthal/angular-react-starter-image:latest
+    ports:
+      - 8080:80
+    restart: always
```

Comment:
Why restart is set to `always`?

Reply:
This option is specifically needed for restarting in case the container suddenly stops (for example, if the program completes its execution normally or crashes)